### PR TITLE
Validate PUMPR

### PR DESCRIPTION
**Pull request by devtonic0**
add PUMPR token

# Validate [{Token Symbol}](https://solscan.io/token/{mint_address})

## Attestations (Please provide links):
- Tweet from your Twitter Account attesting the Mint address, tagging [@JupiterExchange](https://twitter.com/JupiterExchange) and showing community support: https://twitter.com/Pumpr_xyz/status/1730642489483333973
- Coingecko/ CMC URL (If available): https://www.coingecko.com/en/coins/{id}

## Validation (Please check off boxes):
- [x] The metadata provided in the PR matches what is on-chain (Mandatory)
- [x] Does not duplicate the symbol of another token on Jupiter's strict list (If not, review will be delayed)
- [ ] Is Listed on Coingecko / CMC (Optional, but helpful for reviewers)  

## Acknowledgement (Please check off boxes)
- [x] My change matches the format in the file (no spaces between fields).
- [x] My token is already live and trading on Jupiter.
- [x] !!! I read the README section on Community-Driven Validation and understand this PR will be only be reviewed when there is community support on Twitter.

### CODE REVIEW

**Comment by devtonic0** [comment]
added missing X link to tweet.

**Comment by Timoon21** [comment]
Hey @devtonic0 can you add your token at the last line of the file. Not on other line. Thanks.

File: validated-tokens.csv [hunk 1]
```diff
@@ -825,3 +825,4 @@ Ponk,PONK,HeqCcMjmuV5s25J49YiJyT6bD5qWLkP88YPajBySniaV,5,https://bafkreieyllpkeg
 ClownOnSolana,$Clown,V7ntWk4D9Frwaec2CUjMRMMsJG28Cd31HxDKNdP1qjM,9,https://arweave.net/q1EkX68UfbUnvjpA87GaRjGxVaP6f0lYMBeQXmFzN6w,true
 Solabrador,SOBER,2ZE6hSL36e44wP168YMnxrbi1CSCFuD2BJm7NoNHfsmN,9,https://node2.irys.xyz/5xOYKlM26hYXX5wkZoQhc1yw0Bd0Xzoh_SU52TakOZQ,true
 Solanacorn,CORN,6DSqVXg9WLTWgz6LACqxN757QdHe1sCqkUfojWmxWtok,7,https://arweave.net/c_Tbuf4_HSFuogvTV1_FaqqbQipml9YlAETM941s178,true
+Pumpr,PUMPR,PumPRGmZ56t3Vngxo6fCP7ZJQ14oUg3biKxXrEyQBSf,6,https://arweave.net/Drsc75CkUoUY-4c_6iYXTUl2YCMt_0rz6xs8M9Mduq0,false
```
Comment (Timoon21) [inline validated-tokens.csv:828]:
```suggestion
Pumpr,PUMPR,PumPRGmZ56t3Vngxo6fCP7ZJQ14oUg3biKxXrEyQBSf,6,https://arweave.net/Drsc75CkUoUY-4c_6iYXTUl2YCMt_0rz6xs8M9Mduq0,true
```

**Review by Timoon21 — DISMISSED** [review]

**Comment by devtonic0** [comment]
hey guys just looking for an update here, i see @Timoon21  approved the first one 3 days ago, assuming yall are busy as heck

**Pull request merged**